content/en/docs/Reference/image_matrix.md

```diff
@@ -50,8 +50,14 @@ Base images are tagged with specific upstream versions (e.g. Ubuntu 20 LTS is pi
 [c-opensuse-tumbleweed-arm-rpi]: https://quay.io/repository/kairos/core-opensuse-tumbleweed-arm-rpi
 [c-opensuse-tumbleweed-arm-rpi-img]: https://quay.io/repository/kairos/core-opensuse-tumbleweed-arm-rpi-img
 [c-ubuntu]: https://quay.io/repository/kairos/core-ubuntu
+[c-ubuntu-arm-rpi]: https://quay.io/repository/kairos/core-ubuntu-arm-rpi
+[c-ubuntu-arm-rpi-img]: https://quay.io/repository/kairos/core-ubuntu-arm-rpi-img
```

Comment:
Maybe we should add some text above the table to explain the difference between `-rpi` images and `-rpi-img` ones.

Reply:
Good point, added it